accepted/0000-workspaces.md

```diff
@@ -0,0 +1,479 @@
+# npm workspaces
+
+## Summary
+
+Add a set of features to the npm cli that provide support to managing multiple packages from within a singular top-level, root package.
+
+## Motivation
+
+This feature has been requested by the community for a long time. The primary motivation behind this RFC is to fully realize a set of features/functionality to support managing multiple packages that may or may not be used together.
+
+The name “workspaces” is already established in the community with both Yarn and Pnpm implementing similar features under that same name so we chose to reuse it for the sake of simplicity to the larger community involved.
+
+## Detailed Explanation
+
+After sourcing feedback from the community, there are 2 major implementations/changes required in the cli in order to provide the feature set that would enable a better management of nested packages.
+- Install: In a workspaces setup users expect to be able to install all nested packages and perform the associated lifecycle scripts from the root-level, it should also be aware of sibling packages that depend on one another and link them appropriately
+- Run some npm subcommands in the context of each nested workspace
+
+## Rationale and Alternatives
+
+First and foremost there’s the alternative of leaving the problem set for userland to solve, there’s already the very popular project Lerna that provides all of these features.
```

Comment:
```suggestion
First and foremost there’s the alternative of leaving the problem set for userland to solve, there’s already the very popular project Lerna that provides some of these features.
```

I think Lerna is a great tool when you are specifically working in a monorepo.  I hope that we can think beyond repo setup in how users work on projects.  This one is a bit of a nit-pick, so don't put too much weight on the requested change, but I don't want us to fall into the trap of thinking the community has provided everything folks need.  We can and should strive for more than we have today.